CMakeLists.txt

```diff
@@ -1,28 +1,29 @@
 cmake_minimum_required(VERSION 3.3)
 
-project(docks)
-find_package(Qt5Widgets)
+project(KDDockWidgets)
+
+option(OPTION_DEVELOPER_MODE "Developer Mode" OFF)
+option(OPTION_ASAN_SUPPORT "Activate support for using ASAN" OFF)
 
 cmake_policy(SET CMP0020 NEW)
```

Comment:
I think that some policies affect options so maybe it is safer to keep the options after the policy requests ? 

e.g. https://cmake.org/cmake/help/latest/policy/CMP0077.html